hazelcast-spring/src/main/resources/hazelcast-spring-4.0.xsd

```diff
@@ -3457,6 +3458,32 @@
         </xs:attribute>
     </xs:complexType>
 
+    <xs:complexType name="client-metrics">
+        <xs:all>
+            <xs:element name="jmx" type="metrics-jmx" minOccurs="0"/>
+            <xs:element name="collection-frequency-seconds" type="xs:unsignedInt" default="5" minOccurs="0">
+                <xs:annotation>
+                    <xs:documentation>
+                        Sets the metrics collection frequency in seconds.
+                        By default, metrics are collected every 5 seconds.
+                        May be overridden by 'hazelcast.metrics.collection.frequency'
```

Comment:
Probably, we need a separate set of sys props for client overrides, starting with `hazelcast.client.*`.

Reply:
We do need them. Actually, the override for the client config was somehow forgotten. Now added it. Thanks @puzpuzpuz, it was an important catch :100: